tex/generic/pgf/graphdrawing/lua/pgf/gd/trees/doc.lua

```diff
@@ -287,6 +292,9 @@ example
   };
 ]]
 
+[[
+% TODOsp: codeexamples: the next example needs the library `arrows.spaced`
+]]
```

Comment:
This is not a comment but just a literal string.  To make this a comment use
```lua
--[[
% TODOsp: codeexamples: the next example needs the library `arrows.spaced`
--]]
```

Reply:
Now I see. Thanks for this hint. --> Done.